### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by ivy-lli**

### CODE REVIEW

File: playwright/tests/pageobjects/DataClassEditor.ts [hunk 1]
```diff
@@ -6,7 +6,8 @@ import { Settings } from './Settings';
 import { Button } from './abstract/Button';
 import { Table } from './abstract/Table';
 
-const server = process.env.BASE_URL ?? 'http://localhost:8081';
+export const server = process.env.BASE_URL ?? 'http://localhost:8081';
+export const user = 'Developer';
```
Comment (ivy-lmu) [inline playwright/tests/pageobjects/DataClassEditor.ts:10]: I would introduce a global user constant

**Review by ivy-lmu — APPROVED** [review]
👍

File: playwright/tests/pageobjects/DataClassEditor.ts (hunk 1 above)
Comment (ivy-lli) [reply playwright/tests/pageobjects/DataClassEditor.ts:10]: This is a global constant? :)

Comment (ivy-lmu) [reply playwright/tests/pageobjects/DataClassEditor.ts:10]: good for me :D

**Pull request merged**